pennylane/ops/channel.py

```diff
@@ -573,9 +674,23 @@ def __init__(self, *params, wires=None, do_queue=True):
     def num_params(self):
         return 1
 
-    @classmethod
-    def _kraus_matrices(cls, *params):
-        K_list = params[0]
+    @staticmethod
+    def compute_kraus_matrices(K_list):  # pylint:disable=arguments-differ
+        """Kraus matrices representing the QubitChannel channel.
+
+        Args:
+            K_list (list[array[complex]]): list of Kraus matrices
```

Comment:
lol I got very confused here for a moment.

In cases like this, do you think we should just write it like?
```python
class SomeChannel(Channel):
    compute_kraus_matrices = staticmethod(lambda x: x)
```

Reply:
I'd like to have the docstring though :)